src/ansys/templates/python/common/{{cookiecutter.__project_name_slug}}/doc/source/conf.py

```diff
@@ -19,7 +26,12 @@
 {%- endif %}
 copyright = f"(c) {datetime.now().year} ANSYS, Inc. All rights reserved"
 author = "ANSYS, Inc."
+{%- if cookiecutter.__template_name != "doc-project" %}
+release = version = __version__
+{%- elif cookiecutter.__template_name == "doc-project" %}
 release = version = "{{ cookiecutter.__version }}"
+{%- endif %}
+cname = os.getenv("DOCUMENTATION_CNAME")
```

Comment:
```suggestion
cname = os.getenv("DOCUMENTATION_CNAME", "noname.com")
```

Reply:
Since [nocname.com](https://nocname.com) is a real website, let us try to use a different domain. We could redirect to the default [docs.pyansys.com](https://docs.pyansys.com).